test/Tools/Offloader/BufferFuzzy-error.test

```diff
@@ -0,0 +1,113 @@
+#--- source.hlsl
+
+RWStructuredBuffer<float> Out1 : register(u0);
+RWStructuredBuffer<float> Out2 : register(u1);
+
+[numthreads(1,1,1)]
+void main() {
+  Out1[0] = 20.3;
+  Out1[1] = 5.0;
+  Out2[0] = 0.0;
+}
+
+//--- pipeline.yaml
+
+---
+Shaders:
+  - Stage: Compute
+    Entry: main
+    DispatchSize: [1, 1, 1]
+Buffers:
+  - Name: Out1
+    Format: Float32
+    Stride: 4
+    ZeroInitSize: 8
+  - Name: Expected1
+    Format: Float32
+    Stride: 4
+    Data: [ 1.5, 2.5 ]
+  - Name: Out2
+    Format: Float32
+    Stride: 4
+    ZeroInitSize: 4
+  - Name: Expected2
+    Format: Float32
+    Stride: 4
+    Data: [ 0.0000000000000000000000000000000000000112 ] # isDenorm will return true for this value
```

Comment:
For values like this where you really rely on an explicit representation it may be best to encode this as hex with the numeric value in a comment.